### PR TITLE
Add dipole B1 3.2 mrad source point markers

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

**Comment by anacso17** [comment]
@xresende could you please update version in this PR?

File: pymodels/SI_V25_04/segmented_models.py [hunk 1]
```diff
@@ -164,23 +167,33 @@ def dipole_b1(m_accep_fam_name, simplified=False):
...
                 polynom_a=PolyA, polynom_b=PolyB)
         else:
             element = element_type(fam_name)
         model.append(element)
 
+    # --- add source point marker to half-model ---
+    imodel = model[::-1]
+    angles = _np.array([elem.angle for elem in imodel])
+    angles_cumsum = _np.cumsum(angles)
+    src_idx = _np.argmin(_np.abs(angles_cumsum - src_point_angle))
```
Comment (fernandohds564) [inline pymodels/SI_V25_04/segmented_models.py:188]: is this correct? The beamline exit is at 3.2mrad from the start of the field, or from the dipole edge?
How far from 3.2mrad is the result of this `argmin`?

Comment (xresende) [reply pymodels/SI_V25_04/segmented_models.py:188]: @fernandohds564 , inserting the marker this way  it is located after B1 segments with indices 0,1,2,3 and 4, at 3.208 mrad.

Comment (xresende) [reply pymodels/SI_V25_04/segmented_models.py:188]: I committed comments about this.

**Comment by xresende** [comment]
> @xresende could you please update version in this PR?

done, @anacso17

**Review by murilobalves — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

**Pull request merged**